DependencyInjection/SpClient.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Italia\SpidSymfonyBundle\DependencyInjection;
```

Comment:
Move this to a `Service` folder or similar. `DependencyInjection` is a special place in Symfony, better leave it alone